py/client/build.gradle

```diff
@@ -16,49 +12,70 @@ wheel {
-String dockerNetworkName = "pydeephaven-network-${randomSuffix}"
-def createDeephavenGrpcApiNetwork = tasks.register('createDeephavenGrpcApiNetwork', DockerCreateNetwork) {
-    networkName.set dockerNetworkName
-}
-def removeDeephavenGrpcApiNetwork = tasks.register('removeDeephavenGrpcApiNetwork', DockerRemoveNetwork) {
-    networkId.set dockerNetworkName
+configurations {
+    python {}
 }
 
-def createDeephavenGrpcApi = tasks.register('createDeephavenGrpcApi', DockerCreateContainer) {
-    def grpcApiImage = project(':docker-server').tasks.findByName('buildDocker-server')//docker-server-netty
-    dependsOn grpcApiImage, createDeephavenGrpcApiNetwork
-    targetImageId grpcApiImage.getImageId()
-    containerName.set dockerContainerName
-    hostConfig.network.set dockerNetworkName
+dependencies {
+    python project(path: ':proto:proto-backplane-grpc', configuration: 'python')
 }
-def startDeephavenGrpcApi = tasks.register('startDeephavenGrpcApi', DockerStartContainer) {
-    dependsOn createDeephavenGrpcApi
-    containerId.set dockerContainerName
+
+def generatedProtoDir = layout.buildDirectory.dir('generatedProto')
+
+def updatePyNamespaces = tasks.register('updatePyNamespaces', Sync) {
+    from(configurations.python) {
+        // change directory from deephaven to pydeephaven
+        eachFile { it.path = 'pydeephaven/' + it.path.substring('deephaven/'.length()) }
+        // rewrite specific imports/namespaces from deephaven to pydeephaven
+        filter { contents ->
+            def scanner = new Scanner(contents)
+            def lines = []
+            while (scanner.hasNextLine()) {
+                def line = scanner.nextLine();
+                if (line.trim().startsWith('\'__module__\' :')) {
+                    line = line.replace('\'deephaven.', '\'pydeephaven.')
+                } else if (line.startsWith('from deephaven.proto')) {
+                    line = line.replace('from deephaven.proto', 'from pydeephaven.proto')
+                }
+                lines.add(line)
+            }
+            return lines.join(System.lineSeparator());
+        }
+    }
+
+    into generatedProtoDir
 }
-def waitForHealthy = tasks.register('waitForHealthy', WaitForHealthyContainer) {
-    dependsOn startDeephavenGrpcApi
 
-    awaitStatusTimeout.set 20
-    checkInterval.set 100
+def compare = tasks.register('compareProtobuf', DiffTask) {
+    dependsOn updatePyNamespaces
 
-    containerId.set dockerContainerName
+    expectedContents.set generatedProtoDir.get().dir('pydeephaven/proto')
+    actualContents.set layout.projectDirectory.dir('pydeephaven/proto')
+    generateTask.set ':py-client:updateProto'
+    ignore {
+        include '__init__.py'
+    }
 }
-def stopDeephavenGrpcApi = tasks.register('stopDeephavenGrpcApi', DockerRemoveContainer) {
-    dependsOn createDeephavenGrpcApi
-    finalizedBy removeDeephavenGrpcApiNetwork
+// fail a "check" build if these are out of date
+tasks.getByName('quick').dependsOn(compare)
```

Comment:
named

Reply:
I can't find a single instance where we use named with quick, since that requires a .configure closure